CMakeLists.txt

```diff
@@ -57,7 +57,7 @@ if (OPENSN_WITH_CUDA)
 endif()
 
 find_package(HDF5 REQUIRED COMPONENTS C HL)
-find_package(Boost REQUIRED)
+find_package(Boost CONFIG REQUIRED)
```

Comment:
Try `find_package(Boost REQUIRED CONFIG)` instead. That's what works for me in my other project using boost.